Library/Formula/gh-release.rb

```diff
@@ -0,0 +1,15 @@
+class GhRelease < Formula
+  desc "Utility for automating GitHub releases with file uploads"
+  homepage "https://github.com/progrium/gh-release"
+  url "https://github.com/progrium/gh-release/releases/download/v2.2.0/gh-release_2.2.0_darwin_x86_64.tgz"
+  version "2.2.0"
+  sha256 "b272576d9fd6449caa30e64e82f683a0bbc240eadf3be42fad4d8d2c7bd86eef"
+
+  def install
+    bin.install "gh-release"
+  end
+
+  test do
+    system "#{bin}/gh-release", "--help"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!